requirements/requirements-testing.txt

```diff
@@ -1,4 +1,4 @@
 # Pytest for running the tests.
-pytest>=5.4.1,<5.5
-pytest-django>=3.9.0,<3.10
```

Comment:
6.2 deprecates support for py3.5, which is why this fails.  https://github.com/encode/django-rest-framework/pull/7672